client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/SutController.java

```diff
@@ -57,6 +70,22 @@
      */
     private final List<ExtraHeuristicsDto> extras = new CopyOnWriteArrayList<>();
 
+    /**
+     * a map of interface schemas for RPC service under test
+     */
+    private final Map<String, InterfaceSchema> rpcInterfaceSchema = new HashMap<>();
+
+    /**
+     * a map of local auth setup schemas for RPC service under test
+     */
+    private final Map<Integer, LocalAuthSetupSchema> localAuthSetupSchemaMap = new HashMap<>();
```

Comment:
what is the key here?

Reply:
clarification is added as below in code:
 * - key is the index of the auth info which is specified in the driver
 * - value is extracted local auth setup schema